src/runtime_lib/infra_gapbs/intersections.h

```diff
@@ -21,6 +23,12 @@ using namespace std;
 /*
 Runtime library for various set intersection methods
 */
+size_t intersectSortedNodeSetBitset(Bitmap* A, NodeID *B, size_t totalB);
+size_t intersectSortedNodeSetBinarySearch(NodeID *A, NodeID *B, size_t totalA, size_t totalB);
+size_t intersectSortedNodeSetHiroshi(NodeID *A, NodeID *B, size_t totalA, size_t totalB, NodeID dest = (NodeID)INT32_MAX);
```

Comment:
In the comment, add a reference to the original paper.

Reply:
sounds good